documentation/wiki/Binary-Log.md

```diff
@@ -64,6 +64,12 @@ Once you have the `StructuredLogger.dll` on disk you can pass it to MSBuild like
 You can use the MSBuild Structured Log Viewer tool to view `.binlog` files:
 http://msbuildlog.com
 
+# Collecting binary logs from Visual Studio builds
+
+If you need to capture a binary log from within Visual Studio, instead of on the command line, you'll need a Visual Studio plugin: https://marketplace.visualstudio.com/items?itemName=VisualStudioProductTeam.ProjectSystemTools
```

Comment:
```suggestion
If you need to capture a binary log in Visual Studio, instead of the command line, you'll need a Visual Studio plugin: https://marketplace.visualstudio.com/items?itemName=VisualStudioProductTeam.ProjectSystemTools
```